src/CodeGen_ARM.cpp

```diff
@@ -77,6 +81,12 @@ class CodeGen_ARM : public CodeGen_Posix {
     bool neon_intrinsics_disabled() {
         return target.has_feature(Target::NoNEON);
     }
+
+    bool isFloat16AndHasFeature(const Type &t) const {
```

Comment:
Halide convention would be to name this `is_float16_and_has_feature`

Reply:
Done.